### PR TITLE
[Easy] Static OP Updates

### PR DESCRIPTION
**Pull request by MSilb7**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/tokens/optimism/tokens_optimism_erc20_curated.sql [hunk 1]
```diff
@@ -394,6 +394,7 @@ WITH raw_token_list AS (
     ,(0x3a18dcc9745edcd1ef33ecb93b0b6eba5671e7ca, 'KUJI', 6,  'underlying')
     ,(0xdC6fF44d5d932Cbd77B52E5612Ba0529DC6226F1, 'WLD', 18, 'underlying')
     ,(0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487, 'GRAI', 18, 'underlying')
+    ,(0x50Bce64397C75488465253c0A034b8097FeA6578, 'HAN', 18, 'underlying')
```
Comment (Hosuke) [inline models/tokens/optimism/tokens_optimism_erc20_curated.sql:397]: This address is already added:
https://github.com/duneanalytics/spellbook/blob/35a30c19b2ee2985d344fb6b8a104a8a123f8c8c/models/tokens/optimism/tokens_optimism_erc20_curated.sql#L245

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/tokens/optimism/tokens_optimism_erc20_curated.sql (hunk 1 above)
Comment (MSilb7) [reply models/tokens/optimism/tokens_optimism_erc20_curated.sql:397]: ah yeah merge error. resolving!

**Comment by Hosuke** [comment]
These two lines are causing the test failure(duped address):

https://github.com/duneanalytics/spellbook/blob/7b01636ae7ee1e666b440083a817c36d3163eb89/models/addresses/optimism/addresses_optimism_grants_funding.sql#L73

https://github.com/duneanalytics/spellbook/blob/7b01636ae7ee1e666b440083a817c36d3163eb89/models/addresses/optimism/addresses_optimism_grants_funding.sql#L169

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @MSilb7

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**